src/components/select/select.component.ts

```diff
@@ -301,7 +302,8 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
         }
 
         // when the user types and the value is not empty then we should open the dropdown
-        if (event.keyCode !== ESCAPE) {
+        if (event.keyCode !== ESCAPE && !this._valueEmpty) {
+            this._valueEmpty = true;
             // open the dropdown once the filter debounce has elapsed
             this.filter$.pipe(take(1), takeUntil(this._onDestroy))

```

Comment:
I think the issue is with the filter subscription in here. I added this in as a temporary fix yesterday to get it working, but meant to move this, I will help you make this fix after standup

Reply:
updated